pages/common/csvkit.md

```diff
@@ -0,0 +1,21 @@
+# csvkit
+
+> Manipulation toolkit for CSV files.
+> For more information see commands: `csvclean`, `csvcut`, `csvformat`, `csvgrep`, `csvlook`, `csvpy`, `csvsort`, `csvstat`.
+> More information: <https://csvkit.readthedocs.io/en/0.9.1/cli.html>.
+
+- Run a command on a CSV file with the non default delimiter:
+
+`{{cmd}} -d {{delimiter}} {{filename.csv}}`
```

Comment:
I'm a bit confused on `{{cmd}}` here. Generally, a each command should have it's own tldr page. If there are multiple related commands, a description line is fine - but perhaps we could put the command name here?
```suggestion
`csvkit -d {{delimiter}} {{filename.csv}}`
```

The same goes for the examples below.